service-base-quarkus/pom.xml

```diff
@@ -149,6 +155,14 @@
   <profiles>
     <profile>
       <id>jboss-logmanager</id>
+      <!-- alternative until the problem with jboss-logmanager-embedded is fixed -->
+      <dependencies>
+        <dependency>
+          <groupId>org.jboss.logmanager</groupId>
+          <artifactId>jboss-logmanager</artifactId>
```

Comment:
I had also tried to replace the embedded logmanager with the real thing. However, the *real thing* doesn't contain the `org.jboss.logmanager.formatters.ColorPatternFormatter` which is used by Quarkus during startup :-(

Reply:
Ah, I remember now hitting that one at some point as well.